hazelcast/src/main/java/com/hazelcast/internal/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -338,9 +338,9 @@ public void memberAdded(MemberImpl member) {
         logger.fine("Adding " + member);
         lock.lock();
         try {
-            if (!member.localMember()) {
+//            if (!member.localMember()) {
```

Comment:
yes, local member check removed for the case lite member is master case. I forgot to remove commented section.